### PR TITLE
Themes: update search suggestions styling

### PR DESCRIPTION
**Pull request by folletto**
Updated the suggestions styling. Still not identical to the designs, but incrementally closer.

PR status:

screen shot 2016-10-26 at 02 08 42

screen shot 2016-10-26 at 02 08 57

### CODE REVIEW

**Comment by folletto** [comment]
Note that I also changed the logic for the visualization of the tokens in the welcome dropdown in such a way that the styling (colors) is done entirely in CSS.

**Comment by folletto** [comment]
Tested on Chrome, Safari, Firefox, Edge, IE — all seems ok.

Edge and IE exhibit strange bugs in selecting and cursor position, but I can replicate on staging so it's not a regression of this PR.

**Comment by ockham** [comment]
> Note that I also changed the logic for the visualization of the tokens in the welcome dropdown in such a way that the styling (colors) is done entirely in CSS.

Cool :smile: https://github.com/Automattic/wp-calypso/pull/8282#discussion_r84674597

**Comment by ockham** [comment]
UX question -- the row are pretty high. My 15" MBP is pretty much filled up with as little as about 7 rows (plus some headers):

image

File: client/my-sites/themes/themes-magic-search-card/style.scss [hunk 1]
```diff
@@ -136,43 +136,62 @@
...
+	color: rgb( 102, 195, 126 );
+}
+.themes-magic-search-card__welcome-taxonomy-type-layout {
+	color: rgb( 242, 134, 10 );
+}
+.themes-magic-search-card__welcome-taxonomy-type-subject {
+	color: $gray;
+}
+.themes-magic-search-card__welcome-taxonomy-type-style {
+	color: rgb( 241, 56, 59 );
```
Comment (ockham) [inline client/my-sites/themes/themes-magic-search-card/style.scss]: We currently have one more taxonomy, _Picks_, which right now gets the fallback color and "tag" Gridicon. Do we want to style it more individually?

**Comment by ockham** [comment]
Code looks good and tests well. I didn't really review the SASS styling, but the UI looks good, so let's :shipit:

**Comment by ockham** [comment]
I'll merge it -- we can iterate on later.

File: client/my-sites/themes/themes-magic-search-card/style.scss (hunk 1 above)
Comment (folletto) [inline client/my-sites/themes/themes-magic-search-card/style.scss]: Good point, and it's also a good test for the refactoring. I'll follow up.

**Comment by folletto** [comment]
> UX concern -- the rows are pretty high. My 15" MBP is pretty much filled up with as little as about 7 rows (plus some headers):

Yeah, probably needs tweaking. The current height tries to match other heights elsewhere, but you're right might be an issue.

Good call on merging and follow up ;)

**Pull request merged**